examples/network_connectivity_center/main.tf

```diff
@@ -23,16 +23,32 @@ module "network_connectivity_center" {
   ncc_hub_labels = {
     "module" = "ncc"
   }
+  ncc_hub_preset_topology = "STAR"
```

Comment:
@paulwilljones this looks fine. Can you plz add another module call which creates a hub without passing `ncc_hub_preset_topology` and `ncc_groups`? Just want to cover scenario where the values are absent. we dont need spoke in that hub.